modules/nw-sriov-example-dpdk-line-rate.adoc

```diff
@@ -0,0 +1,279 @@
+     net:
+        userLevelNetworking: true
+     numa:
+       topologyPolicy: "single-numa-node"
+     nodeSelector:
+       node-role.kubernetes.io/worker-cnf: ""
+----
+<1> If hyperthreading is enabled on the system, allocate the symbolic link to the same CPU group (isolated/reserved). If the system contains multiple NUMAs, allocate CPUs from both NUMAs to both groups. You can also use the Performance Profile Creator for this task: see "Creating a performance profile" for more information.
+<2> You can also specify a list of devices that will have the queues set to the reserved CPU count. See "Reducing NIC queues using the Node Tuning Operator" for more information.
+<3> Allocate the number and size of hugepages needed. You can specify the NUMA for the hugepages. By default, the system allocates an even number to every NUMA on the system. If needed, you can request the use of a realtime kernel for the nodes. See "Provisioning a worker with real-time capabilities" for more information.
+. Save the `yaml` file as `mlx-dpdk-node-policy.yaml`.
+. Apply the performance profile.
+
+== Configuring an SR-IOV Network Operator
+The SR-IOV network operator is used to allocate and configure Virtual Functions from SR-IOV-supporting Physical Function NICs on the nodes.
+
+For more information on deploying the operator, see "Installing the SR-IOV Network Operator".
+For more information on configuring an SR-IOV network device, see "Configuring an SR-IOV network device".
+
+There are some differences between running DPDK workloads on Intel VFs or Mellanox VFs. We will look at both cases.
+
+The following is an example of an `sriovNetworkNodePolicy` object used to run DPDK applications on Intel NICs:
+[source,yaml]
+----
+    apiVersion: sriovnetwork.openshift.io/v1
+    kind: SriovNetworkNodePolicy
+    metadata:
+     name: dpdk-nic-1
+     namespace: openshift-sriov-network-operator
+    spec:
+     deviceType: vfio-pci
+     needVhostNet: true
+     nicSelector:
+       pfNames: ["ens3f0"]
+     nodeSelector:
+       node-role.kubernetes.io/worker-cnf: ""
+     numVfs: 10
+     priority: 99
+     resourceName: dpdk_nic_1
+    ---
+    apiVersion: sriovnetwork.openshift.io/v1
+    kind: SriovNetworkNodePolicy
+    metadata:
+     name: dpdk-nic-1
+     namespace: openshift-sriov-network-operator
+    spec:
+     deviceType: vfio-pci <1>
+     needVhostNet: true <2>
+     nicSelector:
+         pfNames: ["ens3f1"]
+     nodeSelector:
+     node-role.kubernetes.io/worker-cnf: ""
+     numVfs: 10
+     priority: 99
+     resourceName: dpdk_nic_2
+
+----
+<1> For Intel NICs, `deviceType` must be `vfio-pci`.
+<2> If kernel communication with DPDK workloads is required, add `needVhostNet: true`. This will mount the `/dev/net/tun` and `/dev/vhost-net` devices into the container so the application can create a tap device and connect the tap device to the DPDK workload.
+
+== Using `sriovNetworkNodePolicy` to run DPDK applications on Mellanox NICs
```

Comment:
Please add the same title for Intel NICs or remove it here

Reply:
Removed "Mellanox" from title.

Reply:
please remove the  whole line

Reply:
Removed the heading.